stdlib/time.pyi

```diff
@@ -32,39 +33,30 @@ if sys.version_info >= (3, 8) and sys.platform == "darwin":
 if sys.version_info >= (3, 9) and sys.platform == "linux":
     CLOCK_TAI: int
 
-class _struct_time(NamedTuple):
-    tm_year: int
-    tm_mon: int
-    tm_mday: int
-    tm_hour: int
-    tm_min: int
-    tm_sec: int
-    tm_wday: int
-    tm_yday: int
-    tm_isdst: int
+_ElevenTuple = Tuple[int, int, int, int, int, int, int, int, int, str, str]
+
+# Constructor takes an iterable of any type, of length between 9 and 11 elements.
+@final
+class struct_time(structseq[Union[Any, int]], _ElevenTuple):
```

Comment:
@JukkaL, I'm not sure inheriting from a fixed-length tuple is the right solution for this one specifically, since it can be a tuple of length 9, 10 or 11, depending on how long the argument passed to the constructor is. Maybe inheriting from a variable-length tuple would be better for this one specifically? Also interested in @Akuli's thoughts on this.

Reply:
It seems to always behave like a tuple of 9 elements:

```
>>> list(time.struct_time((1,2,3,4,5,6,7,8,9,10,11)))
[1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Reply:
Ahh, that simplifies things, thanks.

Reply:
Seems like the same goes for `os.stat_result`, too -- it always behaves like a tuple of length 10, even though the constructor takes an iterable of variable length.